thoth/storages/graph/postgres.py

```diff
@@ -5221,15 +5221,23 @@ def _python_interpreters_sync_analysis_result(
             )
 
     @staticmethod
-    def _package_extract_get_env_vars(document: Dict[str, Any]) -> Dict[str, str]:
-        """Obtain Thoth specific environment variables present in the container image."""
+    def _package_extract_get_env_vars(
+        document: Dict[str, Any],
+    ) -> Tuple[Dict[str, str], Dict[str, str]]:
+        """Obtain environment variables present in the container image."""
         result = {}
+        result_no_thoth = {}
+
         for entry in (document["result"].get("skopeo-inspect") or {}).get("Env") or []:
             if entry.startswith(("THAMOS_", "THOTH_")):
```

Comment:
Any reason to keep them separate?

```suggestion
            if entry.startswith(("THAMOS_", "THOTH_")) or entry in ("IMAGE_NAME", "IMAGE_VERSION"):
```

Reply:
just because I thought it was more clear :) but we can do as you suggest instead of creating another dict, thanks!!